scripts/pretest.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# When running `npm test`, this script is intended to run during pretest step.
+
+WS=$(pwd)                          # the protractor directory
+$WS/bin/webdriver-manager update   # updates the local webdriver manager
+$WS/node_modules/.bin/jshint lib spec scripts
+cd $WS/website && npm install      # installs dependencies for the website
```

Comment:
These are required. `bin/webdriver-manager update` installs the `selenium/chromedriver` for the direct connect test. Installing the website's dev dependencies installs marked and lodash locally which are required to pass the unit test `website/docgen/spec/*.js`